src/utils/normalizeScreenshot.js

```diff
@@ -18,16 +19,19 @@ async function normalizeRetinaScreenshot(browser, screenDimensions, base64Screen
     return base64Screenshot;
 }
 
-async function normalizeIOSScreenshot(browser, screenDimensions, base64Screenshot) {
-  const toolbarHeight = 44; // bottom toolbar has always a fixed height of 44px
-  const addressbarHeight = 44; // bottom toolbar has always a fixed height of 44px
+async function normalizeMobileScreenshot(browser, screenDimensions, base64Screenshot) {
+  const mobileFixedHeights = (browser.isAndroid) ? fixedHeights.android : fixedHeights.iOS;
+  const toolbarHeight = mobileFixedHeights.toolbarHeight;
+  const addressbarHeight = mobileFixedHeights.addressbarHeight;
```

Comment:
The issue is that toolbar appears only in some cases. Is there a way to get the y offset of the viewport from the top of the screen? If so then I can calculate whether the footer appears. I think if I use `screenHeight - innerHeight` and the footer is there, it will be the height of the address bar + the toolbar.